partner_risk_insurance/tests/test_partner_risk_insurance.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2018 Tecnativa - Sergio Teruel
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo.tests import TransactionCase, tagged
+
+
+@tagged("post_install", "-at_install")
+class TestSaleOrderLineInput(TransactionCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.partner = cls.env["res.partner"].create(
+            {
+                "name": "Test",
+                "company_credit_limit": 100.00,
+                "insurance_credit_limit": 50.00,
+            }
+        )
+
+    def test_credit_limit(self):
+        self.assertEqual(self.partner.credit_limit, 150.00)
```

Comment:
It seems that the compute is not being triggered, so the test is failing.

Reply:
Actually the problem here is that the `credit_limit` field (https://github.com/odoo/odoo/blob/16.0/addons/account/models/partner.py#L550) and its own logic are added in `account`. I am not very clear what would be the step to follow in this case (deform the standard operation, rename the field or others).

Reply:
Ping @carlosdauden